pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -170,6 +172,22 @@ func (pas *PodAutoscalerStatus) IsInactive() bool {
 	return pas.GetCondition(PodAutoscalerConditionActive).IsFalse()
 }
 
+// HasBeenActive returns true if the pod autoscaler has reached its initial scale.
+func (pas *PodAutoscalerStatus) HasBeenActive() bool {
+	if val, ok := pas.Annotations[initiallyActiveAnnotation]; !ok || val != "true" {
```

Comment:
would you rather care for the time when it happened? Might be useful in future and it'll be harder to change later.

Reply:
Hmm if we want to know how long it’s been “initially active”, making this a condition is probably better. Also if we write a time stamp here for the annotation, table tests will be difficult.

Reply:
🤷 we can keep true. May be we don't need when.
I am fine with status as well. But I think it should be informational, rather than gating.